lua/registereditor/init.lua

```diff
@@ -0,0 +1,15 @@
+local internals = require("registereditor.internals")
+
+local M = {}
+
+local function set_up_user_commands()
```

Comment:
I'd prefer the name to be `setup_...` instead of `set_up...`

Agreed?

Reply:
Yep, changed it.